src/commands/readability.ts

```diff
@@ -0,0 +1,143 @@
+  const desiredText: string[] = [];
+  visit(rst, (node) => {
+    switch (node.type) {
+      case "title": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        let text = textNodes.map((textNode) => textNode.value).join("");
+        // If the last character of the title is not a period, add one.
+        // Missing periods will negatively impact readability.
+        const lastTitleChar = text.charAt(text.length - 1);
+        if (lastTitleChar != ".") {
+          text = text + ".";
+        }
+        desiredText.push(text + "\n");
+        break;
+      }
+      // TODO: Look into whether newlines in the paragraphs are inflating 
+      // readability scores to be artificially high.
+      // If so, remove newlines in paragraphs and directives nodes.
+      case "paragraph": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        const text = textNodes.map((textNode) => textNode.value).join("");
+        // Remove the <some-anchor-tag> markup that the rST parsing leaves 
+        // in the plain text, as this skews readability.
+        const unwantedText = new RegExp("<.*?>");
+        desiredText.push(text.replace(unwantedText, ""));
+        break;
+      }
+      // TODO: Clean up directive output. Check for tables, code blocks,
+      // and other unwanted elements that will skew readability scores.
+      case "directive": {
+        const textNodes = findAll(node, (n) => n.type === "text");
+        const text = textNodes.map((textNode) => textNode.value).join("");
+        // Remove the <some-anchor-tag> markup that the rST parsing leaves 
+        // in the plain text, as this skews readability.
+        const unwantedText = new RegExp("<.*?>");
+        desiredText.push(text.replace(unwantedText, ""));
+        break;
+      }
+    }
+  });
+  return desiredText;
+};
+
+// TODO: Replace source constants in the files. If we're using them, they skew readability.
+const replaceSourceConstants = (
+  s: string,
+  constants: Record<string, string>
+): string => {
+  return Object.keys(constants).reduce(
+    (acc, cur) => acc.replace(`{+${cur}+}`, constants[cur]),
+    s
+  );
+};
+
+const getReadabilityText = async (args: {
+  inputPath: string;
+  snootyConfig: SnootyConfig;
+}): Promise<void> => {
+  const { inputPath, snootyConfig } = args;
+  const outputPath = path.join("output", inputPath);
+  const outputDir = path.dirname(outputPath);
+  const rawText = await fs.readFile(inputPath, "utf8");
+  const document = new MagicString(rawText);
+  const rst = restructured.parse(document.original, {
+    blanklines: true,
+    indent: true,
+    position: true,
+  });
+  const scorableText = getText({
+    inputPath,
+    document,
+    rst,
+    snootyConfig,
+  });
+  console.log(`Creating ${outputPath}`);
+  console.log(`Output directory: ${outputDir}`);
+  await fs.mkdir(outputDir, { recursive: true });
+  await fs.writeFile(outputPath, scorableText.join("\r\n"), "utf8");
```

Comment:
Why use that newline type?
```suggestion
  await fs.writeFile(outputPath, scorableText.join("\n"), "utf8");
```